src/main/kotlin/Application.kt

```diff
@@ -0,0 +1,27 @@
+import calcualator.Calculator
+import calcualator.Operator
+import java.lang.IllegalArgumentException
+
+class Application {
+    fun calculate(context: String): Calculator {
+        if (context.isBlank()) {
+            throw IllegalArgumentException("빈 문자를 입력할 수 없습니다.")
+        }
+
+        var result = Calculator(INIT_NUMBER)
+        var beforeOperator: Operator = Operator.ADD
+        context.split(" ")
+            .forEachIndexed { index, s ->
+                if (index % 2 == 0) {
+                    result = result.operate(beforeOperator, Calculator(s))
+                } else if (index % 2 == 1) {
+                    beforeOperator = Operator.of(s)
+                }
+            }
+        return result
+    }
+
+    companion object {
+        const val INIT_NUMBER = "0"
```

Comment:
상수화 좋습니다! 👍👍

단 INIT_NUMBER는 Application에서만 사용할텐데, 접근제어자를 변경해주는 것도 좋지 않을까요?

```suggestion
        private const val INIT_NUMBER = "0"
```